cloud

```diff
@@ -2017,7 +2025,8 @@ function execute_command() {
         sections_run "$sections" submitters_withdraw \
             "$project" "$new_topic" "${submitters[@]}"
         sections_run "$sections" scheduler_withdraw "$project" "$prefix"
-        sections_run "$sections" cloud_functions_withdraw "$project" "$prefix"
+        sections_run "$sections" cloud_functions_withdraw "$project" "$prefix" \
+            --cache-redirect-function-name="$cache_redirect_function_name"
```

Comment:
I don't see the `cloud_functions_withdraw` function making use of this new argument whatsoever.